libs/application/templates/parental-leave/src/fields/Duration/index.tsx

```diff
@@ -146,29 +142,36 @@ const Duration: FC<FieldBaseProps> = ({ field, application }) => {
                     parentalLeaveFormMessages.shared.months,
                   ),
                 }}
-                rangeDates={{
-                  start: {
-                    date: formatDateFns(currentStartDateAnswer),
-                    message: formatMessage(
-                      parentalLeaveFormMessages.shared.rangeStartDate,
-                    ),
-                  },
-                  end: {
-                    date: formatDateFns(chosenEndDate),
-                    message: formatMessage(
-                      parentalLeaveFormMessages.shared.rangeEndDate,
-                    ),
-                  },
-                }}
+                rangeDates={
+                  currentIndex === 0 &&
+                  answers.firstPeriodStart !==
+                    StartDateOptions.ACTUAL_DATE_OF_BIRTH
+                    ? {
+                        start: {
+                          date: formatDateFns(currentStartDateAnswer),
+                          message: formatMessage(
+                            parentalLeaveFormMessages.shared.rangeStartDate,
+                          ),
+                        },
+                        end: {
+                          date: formatDateFns(chosenEndDate),
+                          message: formatMessage(
+                            parentalLeaveFormMessages.shared.rangeEndDate,
+                          ),
+                        },
+                      }
+                    : undefined
+                }
                 currentIndex={chosenDuration}
                 onChange={(selectedMonths: number) => {
                   clearErrors(id)
-                  const newEndDate = addMonths(
-                    parseISO(currentStartDateAnswer),
-                    selectedMonths,
+
+                  const newEndDate = formatISO(
+                    addMonths(parseISO(currentStartDateAnswer), selectedMonths),
```

Comment:
For future PR: 
I think we might still have a bug here, where it won't reflect half months, right? 
Since `addMonths` floors the value `Positive decimals will be rounded using Math.floor`
Related:
https://github.com/island-is/island.is/pull/3692/files

Reply:
Yes, we will update everything at once when VMST has finalized the API to calculate all the dates

Reply:
It's part of this ticket and all the tickets that depends on it https://app.asana.com/0/1182378413629561/1200358543222708